src/components/widget/nav/NavItem.vue

```
@@ -1,32 +1,58 @@
+    <StatusBadge
+      v-if="badge !== undefined"
+      :label="String(badge)"
+      severity="contrast"
+      variant="circle"
+      class="ml-auto"
+    />
   </div>
 </template>
 
 <script setup lang="ts">
+import { computed, ref } from 'vue'
```

ESLint

[error] Unable to resolve path to module 'vue'.

(import-x/no-unresolved)